packages/controller/src/tasks/syncTenants.ts

```diff
@@ -34,12 +34,16 @@ function* setDefaultAlertmanagerConfigIfEmpty(tenant: string) {
       }
     });
 
-    if (res.data.length) {
+    if (res.status === 200) {
       // Already has config set
       return;
     }
   } catch (err) {
     if (!err.response) {
+      log.error(
```

Comment:
surprised that this is axios again, and not the `got`-based httpcl -- I see you switched here: https://github.com/opstrace/opstrace/pull/882. Don't want to consume too much time on this, but I think it's important to understand why you did the switch back to axios. Don't want to make all the points again; but a super quick reminder: with `httpcl` we get custom logging and retrying behavior that is _super_ valuable. Also, connect() and recv() timeout control that axios will never be able to provide, ...

Reply:
I really wanted to use the httpcl and tried to use it! But for some reason, I just couldn't get it to work and I never managed to get to the bottom of why. Opted to just move forward with what did work

Reply:
Oh, I didn't see this response before @MatApple. Thanks for the follow-up and we'll get there :).